### PR TITLE
style: run prettier on code writing guidelines

### PR DESCRIPTION
**Pull request by nschonni**
Tried a small change to see how prettier will treat the style guide. Had to use some ignores, but I don't like that it didn't like the ignore directives being indented around the code fences

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/css/index.md [hunk 1]
```diff
@@ -172,12 +172,14 @@ In a stylesheet that contains [media query](/en-US/docs/Web/CSS/Media_Queries/Us
 
   Not this: <!--I thought this is the preferred style-->
 
+<!-- prettier-ignore-start -->
```
Comment (nschonni) [inline files/en-us/mdn/writing_guidelines/writing_style_guide/code_style_guide/css/index.md:175]: This doesn't work at the same indent level, but then breaks the list format when it isn't

**Comment by Josh-Cena** [comment]
This is the part where I had been talking about forking the Prettier parser some time ago. We can configure it to ignore certain meta strings like `example-bad`—worse case, we use `html-nolint` and `css-nolint`.

**Comment by nschonni** [comment]
OK, I just pulled out the ignores for now. This partially formats the examples so they'll look like the prettier style for the code examples at least

**Review by wbamberg — APPROVED** [review]
👍  thank you @nschonni !

**Pull request merged**